### PR TITLE
clarify docs for add-online-expansion-handler

### PR DESCRIPTION
**Pull request by maueroats**
rephrase docstring for `add-online-expansion-handler` and `add-online-expansion-monitor`

### CODE REVIEW

File: drracket/drracket/tool-lib.rkt [hunk 1]
```diff
@@ -238,8 +238,8 @@ all of the names in the tools library, for use defining keybindings
   @{Registers a pair of procedures with DrRacket's online expansion machinery. 
     (See also @racket[drracket:module-language-tools:add-online-expansion-monitor].)
     
-    The first two arguments name a procedure in a module that is loaded by 
-    @racket[dynamic-require] is specially designed separate @racket[place].
+    The procedure @racket[id] from @racket[mod-path] is loaded by
```
Comment (rfindler) [inline drracket/drracket/tool-lib.rkt:241]: This needs to be @racket[_id] to get the right typesetting.

File: drracket/drracket/tool-lib.rkt [hunk 2]
```diff
@@ -238,8 +238,8 @@ all of the names in the tools library, for use defining keybindings
   @{Registers a pair of procedures with DrRacket's online expansion machinery. 
     (See also @racket[drracket:module-language-tools:add-online-expansion-monitor].)
     
-    The first two arguments name a procedure in a module that is loaded by 
-    @racket[dynamic-require] is specially designed separate @racket[place].
+    The procedure @racket[id] from @racket[mod-path] is loaded by 
+    @racket[dynamic-require] into a specially designed separate @racket[place].
```
Comment (rfindler) [inline drracket/drracket/tool-lib.rkt:242]: Maybe something like `@tech[#:doc '(lib "scribblings/reference/reference.scrbl")]{place}` would be better than `@racket[place]`?

**Comment by rfindler** [comment]
I made two little comments -- happy to make those changes myself if you'd prefer and then just merge.

Thanks!

**Comment by samth** [comment]
@rfindler if you wan to make an actual edit to the code, you can click the `+-` button while you're typing the comment, and offer a suggestion which can be incorporated directly in the web interface.

File: drracket/drracket/tool-lib.rkt (hunk 1 above)
Comment (samth) [reply drracket/drracket/tool-lib.rkt:241]:
```suggestion
    The procedure @racket[_id] from @racket[mod-path] is loaded by
```

File: drracket/drracket/tool-lib.rkt (hunk 2 above)
Comment (rfindler) [reply drracket/drracket/tool-lib.rkt:242]:
```suggestion
    @racket[dynamic-require] into a specially designed separate @tech[#:doc '(lib "scribblings/reference/reference.scrbl")]{place}.
```

**Comment by rfindler** [comment]
Nice! Thanks, @samth . (Someone should build the docs and look at the output before pushing probably, tho.)

File: drracket/drracket/tool-lib.rkt (hunk 2 above)
Comment (samth) [reply drracket/drracket/tool-lib.rkt:242]:
```suggestion
    @racket[dynamic-require] into a specially designed separate @tech[#:doc '(lib "scribblings/reference/reference.scrbl")]{place}.
```

**Comment by maueroats** [comment]
1. Confirmed that the docs render ok.
2. I see what `@racket[_id]` is supposed to do, but in my local install, both `@racket[_id]` and `@racket[id]` produce the same HTML (a `span` with `class="RktVar"`). If you think all of the functions' formal parameters should have underscores in front of their symbols, then this PR could get bigger to make the edits. I'm not undoing your edit, just wondering.

**Comment by rfindler** [comment]
Oh, my mistake on the `_`! I was confused -- `id` is already part of the reader so the rendering is automatic. Sorry about that.

File: drracket/drracket/tool-lib.rkt [hunk 3]
```diff
@@ -238,8 +238,8 @@ all of the names in the tools library, for use defining keybindings
   @{Registers a pair of procedures with DrRacket's online expansion machinery. 
     (See also @racket[drracket:module-language-tools:add-online-expansion-monitor].)
     
-    The first two arguments name a procedure in a module that is loaded by 
-    @racket[dynamic-require] is specially designed separate @racket[place].
+    The procedure @racket[_id] from @racket[mod-path] is loaded by
```
Comment (rfindler) [inline drracket/drracket/tool-lib.rkt:241]:
```suggestion
    The procedure @racket[id] from @racket[mod-path] is loaded by
```

**Pull request merged**